partcad/src/partcad/project_factory_git.py

```
@@ -107,7 +108,9 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
                             branch_name = remote_head.reference.name
                             short_branch_name = branch_name[branch_name.find("/") + 1 :]
                             pc_logging.debug("Refreshing the GIT branch: %s" % short_branch_name)
-                            origin.pull(short_branch_name)
+                            origin.pull(short_branch_name, env={
+                                "GIT_CONFIG_PARAMETERS": git_config_params
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)